app/routes/applications.js

```diff
@@ -15,7 +15,7 @@ module.exports = router => {
     const applicationId = req.params.applicationId
     const application = req.session.data.applications.find(app => app.id === applicationId)
 
-    res.render('application/timeline', {
+    res.render('applications/timeline', {
```

Comment:
you could do: `applications/timeline/show` but you don't have to.

Reply:
Would this be `index` or `list`, rather than `show`, which would be reserved for single timeline events?

Reply:
The complexity of naming things. We'll go with the `show` convention for the time being.